staging/src/k8s.io/pod-security-admission/policy/check_procMount_restricted_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestProcMountRestricted(t *testing.T) {
+	defaultValue := corev1.DefaultProcMount
+	unmaskedValue := corev1.UnmaskedProcMount
+	otherValue := corev1.ProcMountType("other")
+
+	tests := []struct {
+		name          string
+		pod           *corev1.Pod
+		expectReason  string
+		expectDetail  string
+		expectAllowed bool
+	}{
+		{
+			name: "procMount",
+			pod: &corev1.Pod{Spec: corev1.PodSpec{
```

Comment:
also specifically test that this still forbids unmasked procMount for `hostUsers: false` pods

Reply:
oh, I missed the true / false subloop below